radis/lbl/factory.py

```diff
@@ -763,13 +813,14 @@ def eq_spectrum(
             raise ValueError(
                 "Tgas should be float or Astropy unit. Got {0}".format(Tgas)
             )
-        self.input.rot_distribution = "boltzmann"  # equilibrium
-        self.input.vib_distribution = "boltzmann"  # equilibrium
+        if not self.input.isatom:
+            self.input.rot_distribution = "boltzmann"  # equilibrium
+            self.input.vib_distribution = "boltzmann"  # equilibrium
 
-        # Get temperatures
+            # Get temperatures
+            self.input.Tvib = Tgas  # just for info
+            self.input.Trot = Tgas  # just for info
```

Comment:
```suggestion
            self.input.Tvib = None
            self.input.Trot = None
```

Reply:
removed as inapplicable to equilibrium spectra